spec/requests/api/v1/targets/index_spec.rb

```diff
@@ -0,0 +1,27 @@
+describe 'GET api/v1/targets', type: :request do
+  let!(:user)               { create(:user) }
+  let!(:user_targets)       { create_list(:target, 2, user:) }
+  let!(:targets)            { create_list(:target, 4) }
+  let(:first_user_target)   { user_targets.first }
+  subject { get api_v1_targets_path, headers: auth_headers, as: :json }
+
+  it 'returns a successful response' do
+    subject
+    expect(response).to be_successful
+  end
+
+  it 'returns all user targets and only them' do
+    subject
+    expect(json[:targets].pluck(:title)).to match_array(user_targets.pluck(:title))
```

Comment:
is `title` unique attribute? Maybe is more appropriate to use `id`

Reply:
I am agree with you, is better to use `id`